packages/model-viewer/src/test/features/scene-graph/material-spec.ts

```diff
@@ -142,7 +143,10 @@ suite('scene-graph/material', () => {
       await loadModel(HELMET_GLB_PATH);
       // Disabled cutoff value hack see:
       // https://github.com/google/model-viewer/blob/e3a000111980f5cf018c61dec8695463e0d843a0/packages/model-viewer/src/three-components/gltf-instance/ModelViewerGLTFInstance.ts#L210
-      expect(element.model!.materials[0].getAlphaCutoff())
+      expect(element.model!.materials[0]![$correlatedObjects]
+                 ?.values()
+                 .next()
+                 .value.alphaTest)
```

Comment:
@timmmeh I changed this test because I only wanted the -0.5 in three.js and not reflected in the glTF.